route/dashboard.js

```diff
@@ -0,0 +1,176 @@
+
+        const toDate = moment.unix(query.to)
+        const fromDate = moment.unix(query.from)
+
+        // Diff by minute will return 1, if from-to is larget than 1 minute
+        query.days = Math.ceil(toDate.diff(fromDate, 'minutes') / 1440) // 1440 minutes in a day
+
+        // Same day
+        if (query.days === 0) {
+            query.days = 1
+        }
+        next()
+    } catch (err) {
+        next(err)
+    }
+}
+
+function statusData() {
+    const to = Date.now() / 1000
+    const from = to - 3600
+    return db.getData(from, to).then(data => data[data.length - 1])
+}
+
+function metStats(req, res, next) {
+    req.logger.info('Querying met stats')
+    // Query will return accounts where balance > 0
+    const metHolderQuery = {
+        $expr: { $gt: [{ $toDouble: '$balance' }, 0] }
+    }
+
+    Promise.all([
+        marketData.getRate('ethereum'),
+        marketData.getMarketData('metronome'),
+        statusData(),
+        req.model('Account').countDocuments(metHolderQuery)
+    ]).then(([rate, { symbol, value, marketCapUsd, supply }, status, totalMetHolders]) => {
+        const price = {}
+        price.eth = value / rate
+        price.usd = value
+
+        const stats = {}
+        stats.symbol = symbol
+        stats.price = price
+        stats.marketCapUsd = marketCapUsd
+        stats.totalSupply = Math.round(status.totalMET / 1e18)
+        stats.circulatingSupply = supply
+        stats.proceeds = Math.round(status.proceedsBal / 1e18)
+        stats.metHolders = totalMetHolders
+
+        req.logger.verbose('Sending met stats to client')
+        res.send(stats)
+    }).catch(err => next(err))
+}
+
+function auction(req, res, next) {
+    const logQuery = JSON.stringify(req.query)
+    req.logger.info(`Querying auction stats: ${logQuery}`)
+
+    moment.tz.setDefault('UTC')
+    Promise.all([
+        marketData.getRate('ethereum'),
+        statusData(),
+        auctionsStats.getBuyerCount(req.model('Event'), req.query),
+        auctionsStats.getlastAuctionPrices(req.model('Event'))
+    ]).then(([rate, data, buyerCount, { lastOpeningPrice, lastClosingPrice }]) => {
+        const currentPrice = {}
+        currentPrice.eth = data.currentAuctionPrice / 1e18
+        currentPrice.usd = currentPrice.eth * rate
+
+        const price = {}
+        price.currentPrice = currentPrice
+        price.lastOpeningPrice = lastOpeningPrice
+        price.lastClosingPrice = lastClosingPrice
+
+        const stats = {}
+        stats.price = price
+        stats.availability = data.minting
+        stats.totalAuctions = data.currAuction
+
+        const nextMidnight = moment().startOf('day').add(1, 'day')
+        const remainingSeconds = nextMidnight.diff(moment(), 'seconds')
+        const duration = moment.duration(remainingSeconds, 'seconds')
+
+        stats.timeRemaining = `${duration.hours()} hour(s) ${duration.minutes()} minute(s) ${duration.seconds()} second(s)`
```

Comment:
`timeRemaining` should be provided in raw seconds/ms too.

Reply:
Yes, I will change it.